### PR TITLE
Improvements: Bump up confluent version

### PR DESCRIPTION
**Pull request by lciolecki**
Resolved https://github.com/TouK/nussknacker/issues/1035

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -78,11 +78,14 @@ lazy val publishSettings = Seq(
 )
 
 def nussknackerMergeStrategy: String => MergeStrategy = {
+  case PathList(ps@_*) if ps.last == "module-info.class" => MergeStrategy.first //after confluent bump up to 5.5
```
Comment (mproch) [inline build.sbt:81]: please, let's add TODO: figure how how to handle JDK modules
or sth like that,

File: build.sbt [hunk 2]
```diff
@@ -192,7 +195,7 @@ val slickV = "3.3.2"
 val hsqldbV = "2.5.0"
 val postgresV = "42.2.12"
 val flywayV = "6.3.3"
-val confluentV = "5.4.1"
+val confluentV = "5.5.0"
```
Comment (mproch) [inline build.sbt:198]: which kafka version will be used now?

File: engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/ConfluentUtils.scala [hunk 3]
```diff
@@ -3,15 +3,19 @@ package pl.touk.nussknacker.engine.avro.schemaregistry.confluent
...
 import cats.data.Validated
+import io.confluent.kafka.schemaregistry.ParsedSchema
+import io.confluent.kafka.schemaregistry.avro.{AvroSchema, AvroSchemaProvider}
+import org.apache.avro.Schema
 import org.apache.kafka.common.errors.SerializationException
 
 object ConfluentUtils {
-  //Copied from AbstractKafkaAvroSerDe
+
+  private final val ValueSubjectPattern = "(.*)-value".r
```
Comment (Crack) [inline engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/ConfluentUtils.scala:13]: Can we just use `endsWith`, to remove regexp parsing/execution?

Comment (lciolecki) [reply engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/ConfluentUtils.scala:13]: It's not part of my change, but I agree with you: we can cache it. In fact we run it only at nk and once before deploy process.

File: build.sbt (hunk 2 above)
Comment (lciolecki) [reply build.sbt:198]: Kafka version is still 2.2

**Comment by lciolecki** [comment]
There is some problem with Krylo - we're waiting for @arkadius fiexes.

**Pull request merged**